sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -716,4 +721,119 @@ public HttpPipeline getHttpPipeline() {
 
         return protocolLayer.getDigitalTwinModels().updateWithResponseAsync(modelId, updateOperation, context);
     }
+
+    //==================================================================================================================================================
+    // Component APIs
+    //==================================================================================================================================================
+
+    /**
+     * Get a component of a digital twin.
+     * @param digitalTwinId The Id of the digital twin to get the component from.
+     * @param componentPath The path of the component on the digital twin to retrieve.
+     * @return The application/json string representing the component of the digital twin.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<String> getComponent(String digitalTwinId, String componentPath) {
+        return getComponentWithResponse(digitalTwinId, componentPath)
+            .map(response -> response.getValue());
```

Comment:
nit: this can be replaced by a method reference `.map(DigitalTwinsResponse::getValue)`.

Reply:
Ha, too slow. I just figured this out while looking at another PR

Reply:
So I made this change already